aviatrix_ha.py

```diff
@@ -23,6 +23,7 @@
 WAIT_DELAY = 30
 
 INITIAL_SETUP_WAIT = 180
+INITIAL_SETUP_DELAY = 10
```

Comment:
is this enough? do we need to wait longer?

Reply:
Apache would restart in less than 2 seconds. This should be good enough. 

I also ran a lot of tests by not waiting purposefully. We also handle connection errors in the API's that follow.